modules/nw-http-header-configuration.adoc

```diff
@@ -60,8 +76,8 @@ spec:
           set:
             value: SAMEORIGIN
 ----
-
-When both the `IngressController` spec and `Route` spec are configuring the X-Frame-Options header, then the value set for this header at the global level in the Ingress Controller will take precedence, even if a specific route allows frames.
+ifndef::microshift[]
+When both the `IngressController` spec and `Route` spec are configuring the X-Frame-Options header, then the value set for this header at the global level in the Ingress Controller takes precedence, even if a specific route allows frames.
 
 This prioritzation occurs because the `haproxy.config` file uses the following logic, where the Ingress Controller is considered the front end and individual routes are considered the back end. The header value `DENY` applied to the front end configurations overrides the same header with the value `SAMEORIGIN` that is set in the back end:
```

Comment:
This is for the response message, better to change it as the following:
When both the `IngressController` spec and `Route` spec are configuring the X-Frame-Options response header, then the value set for this header at the global level in the Ingress Controller takes precedence, even if a specific route allows frames. for a request header, route spec value will override ingresscontroller value.

Reply:
@ShudiLi I removed this content for MicroShift because it is not relevant. Does your correction apply to OpenShift?